### DISCUSSION TITLE
Static routes don't take precedence over dynamic routes (Nextjs 13.1.5)

### DISCUSSION THREAD

**Discussion by davidebriscese** [vercel/next.js#45944] [Help] [answered]
I have a dynamic route like `/app/admin/[section]/page.jsx` and a static route like `/app/admin/settings.jsx`.

Content of the file `/app/admin/[section]/page.jsx `:
```jsx
export default function Page() {
  return 'dynamic route'
}
```

Content of the file `/app/admin/settings.jsx`:
```jsx
export default function Page() {
  return 'static route'
}
```

Hitting the url `/admin/settings` I would expect to receive the contents of the `settings.jsx` file, but instead it fall into the dynamic route.

This is the folder structure:
![image](https://user-images.githubusercontent.com/13695638/219050858-a1c0b7ce-1503-44e0-8fea-a0b29a3bf4a6.png)

And this is the result if hit localhost:3000/admin/settings:
![image](https://user-images.githubusercontent.com/13695638/219050918-823bb7e4-0525-4e83-bd17-58e1b38b2c46.png)

A different behavior is described [in the documentation](https://nextjs.org/docs/routing/dynamic-routes#caveats):
```
Predefined routes take precedence over dynamic routes, and dynamic routes over catch all routes. Take a look at the following examples:
pages/post/create.js - Will match /post/create
pages/post/[pid].js - Will match /post/1, /post/abc, etc. But not /post/create
pages/post/[...slug].js - Will match /post/1/2, /post/a/b/c, etc. But not /post/create, /post/abc
```

I'm using Nextjs 13.1.5.

**Comment by dan-made** [comment, marked as answer]
settings.jsx isn't a page component, I believe you want settings/page.jsx

**Reply by davidebriscese** [reply]
Correct, my bad. I'm migrating to Nextjs13 and I'm getting confused 😩.
Thank you!
